.gitignore

```diff
@@ -1,3 +1,14 @@
+#Eclipse
```

Comment:
You can ignore these in your local `.git/info/exclude`, you don't need to add these to `.gitignore`

Reply:
That is practical, i didn't know that feature. I will revert the gitignore changes...